src/main/webapp/css/vacation-overview.css

```diff
@@ -108,10 +108,86 @@
   font-size: 12px;
 }
 
+.vacationOverview-table th,
 .vacationOverview-table td {
-  border: 1px solid #FFF;
+  color: #444444 !important;
+  border: 1px solid #ffffff;
+  font-weight: normal;
 }
 
-.vacationOverview-tbody tbody {
-  border: 1px solid #000;
+.vacationOverview-table td {
+  text-align: center;
+}
+
+.vacationOverview-table thead th {
+  padding: 8px;
+  cursor: default !important; /* override $.tablesorter */
+}
+
+.vacationOverview-table tbody th {
+  padding-top: 8px;
+  padding-bottom: 8px;
+  padding-left: 0;
+  padding-right: 16px;
+}
+
+@media print {
+
+  .vacationOverview-table th,
+  .vacationOverview-table td {
+    /*border: none !important;*/
+    border: 1px solid #e2e8f0 !important;
+  }
+
+  .vacationOverview-table thead th {
+    font-weight: 600 !important;
+    padding: 4pt 8pt;
+  }
+  .vacationOverview-table tbody th {
+    font-weight: 300 !important;
+    padding: 4pt 8pt;
+  }
+
+  .vacationOverview-table-legend th {
+    text-align: center;
+    padding: 0 12pt;
+    font-weight: 300 !important;
+  }
+
+  .vacationOverview-day-personal-holiday-status-WAITING {
+    color: #b7791f !important;
```

Comment:
Color differences are hard to see on display (maybe also hard to print?)

Reply:
yep, therefore the text ;-)

Reply:
Text is working on Windows, for some unknown reasen text is not displayed with linux os printing